nextflow.config

```diff
@@ -256,6 +256,13 @@ profiles {
         wave.freeze             = true
         wave.strategy           = 'conda,container'
     }
+    seqera_containers {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
```

Comment:
This should be removed otherwise it will cause the use of a docker container to be converted to singularity one ..

Reply:
also worth adding `wave.build.conda.mambaImage = 'mambaorg/micromamba:1.5.8-lunar'` to prevent containers changes quietly when the base image will change on wave side